test/commands/track/determine_concepts_layout_test.rb

```diff
@@ -28,7 +28,8 @@ def test_layout_with_one_level
             {
               slug: 'basics',
               name: 'Basics',
-              web_url: 'https://test.exercism.io/tracks/ruby/concepts/basics'
+              web_url: 'https://test.exercism.io/tracks/ruby/concepts/basics',
+              tooltip_url: '/tracks/ruby/concepts/basics/tooltip'
```

Comment:
Can we make these URLs, not paths pls? (You'll obviously need to change the code to match -`_path` to `_url`.)

Reply:
Oh, I think I just copied the ruby helper code from your example.  I didn't know that the ending changed things! :laughing: